pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicacmgkehubfeaturemembership/_http.log

```diff
@@ -1,7 +1,7 @@
-GET https://cloudresourcemanager.googleapis.com/v1/projects/example-project-01?alt=json&prettyPrint=false
+GET https://serviceusage.googleapis.com/v1/projects/${projectId}/services?alt=json&fields=services%2Fname%2CnextPageToken&filter=state%3AENABLED&prettyPrint=false
```

Comment:
This looks like the MockGCP GKEHub misses some calls to list services and projects, right?

Reply:
According to the URL, I don't think the list calls should be handled by the MockGKEHub, instead it should be handled by the MockProject.